app/views/overview/_service.html

```diff
@@ -10,38 +10,49 @@
     </div>
   </div>
 </div>
-
 <div ng-attr-row="{{!service ? '' : undefined}}"
      ng-attr-wrap="{{!service ? '' : undefined}}"
-     ng-if="(visibleDeploymentsByConfig | hashSize) || (visibleReplicaSets | hashSize) || (petSetsByService[service.metadata.name] | hashSize) || (monopodsByService[service.metadata.name || ''] | hashSize)"
+     ng-if="(visibleDeploymentsByConfig | hashSize) || (visibleReplicaSetsByDeployment | hashSize) || (petSetsByService[service.metadata.name] | hashSize) || (monopodsByService[service.metadata.name || ''] | hashSize)"
      class="deployment-block"
      ng-class="{
        'no-service': !service,
-       'service-multiple-targets': rcTileCount + (visibleReplicaSets | hashSize) + (petSetsByService[service.metadata.name] | hashSize) + (monopodsByService[service.metadata.name] | hashSize) > 1
+       'service-multiple-targets': rcTileCount + rsTileCount + (petSetsByService[service.metadata.name] | hashSize) + (monopodsByService[service.metadata.name] | hashSize) > 1
      }">
-  <div ng-repeat="(dcName, deployments) in visibleDeploymentsByConfig" class="deployment-tile-wrapper">
+  <div ng-repeat="(dcName, deployments) in visibleDeploymentsByConfig track by dcName" class="overview-tile-wrapper">
     <!-- visible deployments with a dc -->
     <overview-deployment-config ng-if="dcName"></overview-deployment-config>
     <!-- /visible deployments with a dc -->
 
-    <!-- visible deployments without a dc -->
-    <div class="deployment-tile-wrapper" ng-if="!dcName.length" ng-repeat="deployment in deployments | orderObjectsByDate : true track by (deployment | uid)">
-      <overview-replication-controller></overview-replication-controller>
+    <!-- visible replication controllers without a dc -->
+    <div ng-if="!dcName"
+         ng-repeat="set in deployments | orderObjectsByDate : true track by (deployment | uid)"
```

Comment:
you need to fix all the track by references to deployment, they need to be 'set'